src/tsickle_compiler_host.ts

```diff
@@ -27,7 +27,8 @@ import {isDtsFileName} from './tsickle';
 export enum Pass {
   NONE,
   DECORATOR_DOWNLEVEL,
-  CLOSURIZE
+  CLOSURIZE,
+  DECORATOR_DOWNLEVEL_AND_CLOSURIZE,
```

Comment:
Can you add a comment here about how you do both in a single pass, and also that we want to drop the other ones?